include/igl/copyleft/comiso/nrosy.cpp

```diff
@@ -521,65 +447,32 @@ void igl::copyleft::comiso::NRosyField::setConstraintSoft(const int fid, const d
 
 void igl::copyleft::comiso::NRosyField::resetConstraints()
 {
-  using namespace std;
-  using namespace Eigen;
-
-  isHard.resize(F.rows());
-  for(unsigned i=0; i<F.rows(); ++i)
-    isHard[i] = false;
-  hard   = VectorXd::Zero(F.rows());
-
-  wSoft  = VectorXd::Zero(F.rows());
-  soft   = VectorXd::Zero(F.rows());
+  isHard.resize(F.rows(), false);
+  hard   = Eigen::VectorXd::Zero(F.rows());
+  wSoft = Eigen::VectorXd::Zero(F.rows());
+  soft = Eigen::VectorXd::Zero(F.rows());
 }
 
 Eigen::MatrixXd igl::copyleft::comiso::NRosyField::getFieldPerFace()
 {
-  using namespace std;
-  using namespace Eigen;
-
-  MatrixXd result(F.rows(),3);
-  for(unsigned i=0; i<F.rows(); ++i)
+  Eigen::MatrixXd result(F.rows(),3);
+  for(unsigned int i = 0; i < F.rows(); ++i)
```

Comment:
@jdumas I know but my preference is to write this like that. I can revert this.

Reply:
No it's ok, since the file is already inconsistent in its usage of `unsigned` vs `unsigned int` I don't have any strong feeling either way. Just wanted to mention it.